### PR TITLE
add file

### PR DESCRIPTION
**Pull request by tuananhvp25081995**

### CODE REVIEW

**Comment by tuananhvp25081995** [comment]
https://edu-redmine.framgia.vn/issues/3736?issue_count=4&issue_position=1&next_issue_id=3670

**Comment by tuananhvp25081995** [comment]
Ready

File: app/views/static_pages/help.html.erb [hunk 1]
```diff
@@ -1,2 +1,6 @@
-<h1>StaticPages#help</h1>
-<p>Find me in app/views/static_pages/help.html.erb</p>
+<% provide :title, t(".help1") %>
+<div id = "help">
+  <p>
+    <%= link_to t(".please"),Settings.static_pages.help.link1 %> <%= t ".help" %>
```
Comment (huuphuc93) [inline app/views/static_pages/help.html.erb]: sau , cách 1 spaces ra em

File: app/views/static_pages/home.html.erb [hunk 2]
```diff
@@ -1,2 +1,13 @@
...
+<nav class = "group">
+  <ul class="nav navbar-nav navbar-right">
+    <li><%= link_to t(".truyencuoi"), "#" %></li>
+    <li><%= link_to t(".truyenteen"), "#" %></li>
+    <li><%= link_to t(".truyenngungon"), "#" %></li>
+    <li><%= link_to t(".truyenngan"), "#" %></li>
+    <li><%= link_to t(".truyenngontinh"), "#" %></li>
+  </ul>
+  <form class="form-inline my-2 my-lg-0">
+    <input class="form-control mr-sm-2" type="search" placeholder="Search" aria-label="Search">
```
Comment (huuphuc93) [inline app/views/static_pages/home.html.erb]: I18n cho placeholder

**Comment by huuphuc93** [comment]
commented

**Comment by tuananhvp25081995** [comment]
Ready

**Pull request merged**